docker/install.sh

```diff
@@ -1,33 +1,47 @@
 #!/bin/bash
 set -e
 
-# ROS 1 msgs setup
+# Prepare workspace for ROS Noetic dependencies and Autoware messages
+mkdir -p /home/carma/.base-image/ros1_msgs_ws/src/autoware.ai
+
+# Copy Autoware.ai messages into ROS1 workspace
 cp -R /home/carma/autoware.ai/messages /home/carma/.base-image/ros1_msgs_ws/src/autoware.ai/
 cp -R /home/carma/autoware.ai/jsk_common_msgs /home/carma/.base-image/ros1_msgs_ws/src/autoware.ai/
 cp -R /home/carma/autoware.ai/jsk_recognition /home/carma/.base-image/ros1_msgs_ws/src/autoware.ai/
 
-# ROS 2 msgs setup
-mkdir -p ~/.base-image/ros2_msgs_ws/src/autoware.ai
+# ROS1 CARMA message and dependency setup
+cd /home/carma/.base-image/ros1_msgs_ws
+unset ROS_DISTRO
+export ROS_VERSION=1
+colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release --packages-up-to \
+    pcl_msgs \
+    autoware_msgs cav_msgs cav_srvs j2735_v2x_msgs can_msgs carma_debug_msgs autoware_lanelet2_msgs carma_cooperative_perception_interfaces
+
+# ROS2 CARMA message setup
+mkdir -p /home/carma/.base-image/ros2_msgs_ws/src/autoware.ai
 cp -R /home/carma/autoware.ai/messages /home/carma/.base-image/ros2_msgs_ws/src/autoware.ai/
 cp -R /home/carma/autoware.ai/jsk_common_msgs /home/carma/.base-image/ros2_msgs_ws/src/autoware.ai/
 cp -R /home/carma/autoware.ai/jsk_recognition /home/carma/.base-image/ros2_msgs_ws/src/autoware.ai/
 
-# Cleanup autoware repo once messages have been moved
+# Clean up Autoware repository after moving messages
 rm -rf /home/carma/autoware.ai/
 
-# ROS1 message setup
-cd ~/.base-image/ros1_msgs_ws && source /opt/ros/noetic/setup.bash && colcon build --packages-up-to autoware_msgs cav_msgs cav_srvs j2735_v2x_msgs can_msgs carma_debug_msgs autoware_lanelet2_msgs carma_cooperative_perception_interfaces
-
-# ROS2 message setup
-cd ~/.base-image/ros2_msgs_ws && source /opt/ros/humble/setup.bash && colcon build --packages-up-to autoware_msgs autoware_lanelet2_msgs can_msgs carma_debug_ros2_msgs carma_driver_msgs carma_localization_msgs carma_msgs carma_perception_msgs carma_planning_msgs carma_v2x_msgs j2735_v2x_msgs carma_cooperative_perception_interfaces
+cd /home/carma/.base-image/ros2_msgs_ws
+unset ROS_DISTRO
```

Comment:
If we're sourcing humble after this we shouldn't need to unset the ROS distro

Reply:
yeah we probably don't need this step